src/Symfony/Component/HttpFoundation/File/MimeType/ExtensionGuesser.php

```diff
@@ -56,7 +56,7 @@ public static function getInstance()
     /**
      * Registers all natively provided extension guessers.
      */
-    private function __construct()
+    protected function __construct()
```

Comment:
Singletons can be inherited, so child constructors can use this one. And it still back compatible.